### ISSUE TITLE
Restructure Troubleshooting pages to be better consumable

### ISSUE THREAD

**Issue by mydea** [getsentry/sentry-docs#9664]
E.g. here: https://docs.sentry.io/platforms/javascript/troubleshooting/ we have a very long list of things that can be hard to parse.

I would propose to restructure this a bit to be better consumable.

We could split each item into a separate page (this would make it easier to find stuff on Google etc. as well). We could then also aggregate all the items on the "Overview" page of troubleshooting (possibly renaming it to FAQ or something like this?), where we could also present items e.g. as a searchable accordion.

**Comment by Shubhdeep12** [comment]
also how about adding commonly asked questions by devs in Discord?

**Comment by AbhiPrasad** [comment]
@Shubhdeep12 I always liked the idea of linking to GH discussions, and then having some way to promote GH discussion into a troubleshooting topic.

I'd rather that because it's easily searchable via google (discord is hard to index). ideally we can push discord asks into GH as often as possible so that they can be indexed accordingly.

**Comment by stephanie-anderson** [comment]
I think this would have been possible with Giscus, cc @a-hariti @lizokm 
see https://github.com/getsentry/sentry-docs/pull/11132

**Comment by stephanie-anderson** [comment]
Can we transform this so it's read-only and we don't need an authenticated GH user?
